packages/hoppscotch-common/src/components/collections/EditRequest.vue

```diff
@@ -115,6 +115,11 @@ const generateRequestName = async () => {
 
   isGenerateRequestNamePending.value = true
 
+  platform.analytics?.logEvent({
+    type: "EXPERIMENTS_GENERATE_REQUEST_NAME_WITH_AI",
+    platform: "rest",
+  })
```

Comment:
This component is consumed in the `/graphql` page and hence `platform` needs to be specified accepting the context. Leaving a comment here for future reference.

https://github.com/hoppscotch/hoppscotch/blob/40f93fc7f44f25ac607eb7aace16a715f8f42255/packages/hoppscotch-common/src/pages/graphql.vue#L62-L68